lib/enigma.rb

```diff
@@ -1,71 +1,106 @@
 require 'date'
 class Enigma
-  def rand_key
-    rand(99999)
-  end
+  attr_reader :alphabet
 
-  def zero_pad(key)
-    pad_num = key.to_s
-    pad_num.insert(0, "0") until pad_num.size == 5
-    pad_num
+  def initialize
+    @alphabet = Array("a".."z")<< " "
```

Comment:
made alphabet into an instance variable as it should be present in every instance of `Enigma`